### ISSUE TITLE
[Nav EOWG] Standards/Guidelines inconsistent with other nav items

### ISSUE THREAD

**Issue by iamjolly** [w3c/wai-website-design#83]
In the main nav, we have the main, top nav items currently listed as:

- Accessibility Fundamentals
- Plan & Manage
- Design & Develop
- Test & Evaluate
- Teach & Advocate
- Standards/Guidelines

The use of the slash without spacing—and even with spacing—looks and feels inconsistent with our use of the ampersands between grouped terms. It bothers me like a mosquito bite I want to scratch every time I see the nav. That is subjective, however. Others may not share my opinion, and that's OK.

But, I'd like to know **why** we're using the "/" to separate the terms. Possibly there is a better alternative which would address the why of this without making it seem like an error of inconsistency in the main nav.

Are Standards/Guidelines interchangeable terms, like synonyms, so that is why "&" doesn't work here? Or, are they distinct in some way? If they are distinct, even slightly, then perhaps "&" as a separator makes more sense and keeps the item consistent with the other nav items before it.

**Comment by yatil** [comment]
I, too, think it looks weird. It is a bit of a problem that W3C standards are called “recommendations” unless we call them “guidelines” like WCAG :-D 

I think for most users writing “Standards & Guidelines” would be ok. I could even see “W3C WAI Standards” or “Web Accessibility Standards” (probably way too long), but what if someone searches for the Web Content _Guidelines_? They would probably be confused.

This is hard.

**Comment by shawna-slh** [comment]
Yes, we know. We carefully considered this. Basically, "Standards & Guidelines" could communicate to some that the Guidelines are not Standards, and that would be a **very bad thing**!
More in:
* https://lists.w3.org/Archives/Public/public-wai-eo-site/2017Jul/0000.html
* https://lists.w3.org/Archives/Public/public-wai-eo-site/2017Jul/0001.html
* agreed on https://www.w3.org/2017/07/05-eo-minutes.html
* https://lists.w3.org/Archives/Public/public-wai-eo-site/2017Jul/0004.html

@iamjolly , I recommend (pun intended) baking soda or oatmeal or Vit C powder.
If that doesn't work, you can re-open the issue. ;-)

**Comment by iamjolly** [comment]
I'm leaving this closed, but I **highly recommend testing** different variations of this label as part of the usability testing. If Guidelines are, in essence, Standards, then why have both terms? I also disagree, respectfully, with this:

> Basically, "Standards & Guidelines" could communicate to some that the Guidelines are not Standards, and that would be a **very bad thing**!

I can't imagine that the label with an ampersand will be harmful to people's understanding of the content within that section. I think that's a pretty big stretch and highly unlikely. But, it is something we could test for to find out. 

Food for thought for testing:
- Standards/Guidelines (no change)
- Standards & Guidelines (change keeping nav consistent)
- Accessibility Standards (change, using one term)
- Standards (change, with only one term and more compact)

**Comment by shawna-slh** [comment]
Thanks for the additional thoughts.

> If Guidelines are, in essence, Standards, then why have both terms?

See info in links above. Many people know only Web Content Accessibility Guidelines, and thus likely won't think to look under "Standards" for it. I don't think that 8 users is enough to test this.

> I can't imagine that the label with an ampersand will be harmful to people's understanding of the content within that section.

The concern is not so much that they wouldn't understand what content is in that section. It's more the overall issue of not understanding that WCAG = W3C Recommendation = international Web Standard. That is a known issue. (We have that in https://www.w3.org/WAI/intro/w3c-process and I will look at adding it in https://www.w3.org/WAI/intro/components.php )

I'll send ideas for what to ask and what to look for in usability testing to Charlotte (although I don't feel like this is a high priority for lots of focus).

**Comment by shawna-slh** [comment]
Thinking more about this and the number of things that we have for usability testing, I'm not convinced that is an important thing to focus on for usability testing, and we're not planning AB testing, afaik.

I think there is strong support for using both words. We could watch for participants stumbling over "Standards/Guidelines", but even that I'm not sure is a priority.

**Comment by bakkenb** [comment]
+1 to @slhenry.

Let's keep an eye open for comments about this in the usability testing if any should arise. Then we can also watch for comments about this when the new design is unveiled and make edits then if needed.
